unstructured_platform_plugins/etl_uvicorn/errors.py

```diff
@@ -45,15 +41,10 @@ class CatchAllError(BaseError):
 
 
 def wrap_error(e: Exception) -> HTTPException:
-    if isinstance(e, ingest_errors.UserAuthError):
+    if isinstance(e, HTTPException):
```

Comment:
btw this is where its called for HTTPException 
https://github.com/Unstructured-IO/unstructured-platform-plugins/blob/main/unstructured_platform_plugins/etl_uvicorn/api_generator.py#L219
do you think we can also move `status_code=wrap_error(exc).status_code,` -> `status_code=exc.status_code`?

Reply:
tbh we don't need a case for `HTTPException` if we don't call this func for it...

Reply:
hmm no i don't get it tho
if you call `wrap_error(e = HTTPException)` you would get a `CatchAllError(e)` which will convert you status code to 512
which is the case for my ticket Nex-57 we see a lot of 512 with emppty detail